src/deep-security/lib/CredentialsManager.js

```diff
@@ -147,10 +147,10 @@ export class CredentialsManager {
           return callback(true);
         });
       },
-      onDatasetDeleted: (dataset, datasetName, callback) => {
+      onDatasetDeleted: (dataset, datasetName, cb) => {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['cb' is defined but never used.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1509414841/issues/source?bid=2895475&fileBranchId=2980272#l150)